unit-tests/shared/src/test/scala/javalib/util/zip/InflaterTest.scala

```diff
@@ -116,9 +116,11 @@ class InflaterTest {
   def setUp(): Unit = {
     outPutBuff1 = Array[Byte](120, 94, 99, 100, 102, 97, -25, 72, 45, 42, -87,
       52, 5, 0, 6, -12, 2, 17) ++ Array.fill[Byte](483)(0)
-    outPutDiction = Array[Byte](120, 63, 13, 10, 107, 2, 20, 99, 97, 101, 102,
-      74, 76, 98, 99, -25, -32, 100, 40, 54, 46, 47, 2, 13, 112, 2,
-      127) ++ Array.fill[Byte](474)(0)
+    outPutDiction =
+      Array[Byte](120, 63, 13, 10, 107, 2, 20, 99, 97, 101, 102, 74, 76, 98, 99,
+        -25, -32, 100, 40, 54, 46, 47, 2, 13, 112, 2, 127) ++ Array.fill[Byte](
```

Comment:
Can we move the Array.fill to the next line?

Reply:
Ok, that worked and looks nice